### PR TITLE
Change wording on signature page for 3rd party

### PR DESCRIPTION
**Pull request by hartsick**
screen shot 2018-11-12 at 5 35 52 pm

### CODE REVIEW

File: app/decorators/change_report_decorator.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def last_paycheck
   end
 
   def client_name
-    [member&.first_name, member&.last_name].compact.join(" ") if member
+    member&.full_name
```
Comment (ondrae) [inline app/decorators/change_report_decorator.rb]: I never knew about the Safe Navigation Operator. Def going to start using it.

File: spec/features/verification_upload_spec.rb [hunk 2]
```diff
@@ -3,10 +3,25 @@
 feature "Uploading verifications", js: true do
   before do
     visit "/"
+    expect(page).to have_text "Report the change"
```
Comment (ondrae) [inline spec/features/verification_upload_spec.rb]: Are these spec changes mean to be with this pull request?

Comment (hartsick) [reply spec/features/verification_upload_spec.rb]: Yep! As part of this change, we need to make sure we have a member before we skip to where this test skips. Before we didn't. I added some extra text in the test to help us clarify where we are in the flow, too.

File: app/decorators/change_report_decorator.rb (hunk 1 above)
Comment (hartsick) [reply app/decorators/change_report_decorator.rb]: It's relatively new (ruby 2.3.0)! and super useful.

**Pull request merged**